### PR TITLE
docs: add sudo for install commands

### PR DESCRIPTION
**Pull request by caicancai**

### CODE REVIEW

File: docs/src/cn/dev/compile_run.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 假设我们的开发环境是 Ubuntu20.04, 可以执行如下命令来安装所需的依赖。
 
 ```shell
-apt install git curl gcc g++ libssl-dev pkg-config cmake
+sudo apt install git curl gcc g++ libssl-dev pkg-config cmake
```
Comment (jiacai2050) [inline docs/src/cn/dev/compile_run.md:8]: `protobuf-compiler` is also required.

Comment (caicancai) [reply docs/src/cn/dev/compile_run.md:8]: Do you need me to add it in PR?

Comment (caicancai) [reply docs/src/cn/dev/compile_run.md:8]: sudo apt install git curl gcc g++ libssl-dev pkg-config cmake protobuf ?

**Review by jiacai2050 — APPROVED** [review]
LGTM

**Review by tanruixiang — APPROVED** [review]
Thanks for your contribution. LGTM.

**Pull request merged**